Update imagedata.h

stefan123t opened the pull request:
updated the logo with a proper logarithmic spiral cable

stefan123t committed:
Update imagedata.h

stefan123t committed:
Update imagedata.h

leave 25 pixels, i.e. 12.5 top and bottom

stefan123t committed:
add 64x64 pixel version of logo for OLED and Nokia LCD displays

added a 64x64 pixel version of the logo for OLED 128x64 and Nokia LCD displays. 
Reduced structure to 64x64 to conserve half of the PROGMEM space instead of 128x64 pixel size.
May need to be displayed centered on display.

stefan123t committed:
logo 200x200 with 50 pixel white border

stefan123t committed:
logo 200x200 with 30 pixel white border

15 px top and 15 px bottom white border added for Knickohr

stefan123t committed:
logo 200x200 with 25 pixel white border

lumapu commented:
The 64x64 graphic is currently not implemented - am I right?

lumapu commented:
wouldn't it be better to align the logo in ePaper centered as well and remove the 25px white border from the image data? Memory is costly, therefore I would prefer to store only necessary data.

stefan123t commented:
@lumapu yes, these are both valid points.
I can remove the border around the larger 200x200 logo, that would even save some more space left and right too ...
Someone who does the Display stuff would need to align the two versions on the different displays, i.e. OLED, Nokia LCD and ePaper's.
Cheers,
Stefan

stefan123t committed:
reduced logo size, no borders, padding has to be done

As requested by @lumapu I have reduced both logo sizes.
I have added the display size these logos fit and the required left/right and top/bottom padding in a comment each.
You will have to replace all occurrences of logo with logo_spiral_140x150 or implement logo_spiral_52x56 for the OLED and Nokia LCD Displays.

stefan123t commented:
@lumapu I have prepared two Versions for the different displays:
* for OLED, Nokia LCD: **logo_spiral_52x56.png**
   * fits 64x64 or 64x128, etc.
   * padding top/bottom:  4 px 
   * padding left/right 6 px
   ![logo_spiral_52x56](https://github.com/user-attachments/assets/b72b3864-f3eb-45a1-9b12-a0f878fc6009)
* for ePaper **logo_spiral_140x150.png**
   * fits 200x200
   * padding top/botton: 25 px
   * padding left/right: 30 px
   ![logo_spiral_140x150](https://github.com/user-attachments/assets/b7f96137-c5ce-4ef0-b588-2a4131252f38)
